packages/manager/modules/pci/src/projects/project/storages/storage-users.service.js

```diff
@@ -61,7 +61,7 @@ export default class PciStoragesUsersService {
 
   getUsersCredentials(serviceName, usersIds) {
     const promises = usersIds.map((userId) =>
-      this.asCredentials(serviceName, userId),
+      this.asCredentials(serviceName, userId.id),
```

Comment:
**nitpick:** `userId.id` doesn't make sense. Can be renamed to rename.

Reply:
Okay renaming it to user.id